### ISSUE TITLE
How to start it?

### ISSUE THREAD

**Issue by Tangzy7** [clam004/RL-Chat-pytorch#1]
Hi, thanks for your sharing, and I am interested in it. I just have 2 questions

1、I am interested in the way you calculate the loss = loss*reward. Could you share the paper you referred?
2、What are the difference between RL and RL2.ipynb
